### PR TITLE
Introduce Docker

### PR DESCRIPTION
**Pull request by potsbo**
## WHY

In some situations, running with Docker is highly maintainable.

## WHAT

- [x] Add Dockerfile
- [x] Update .travis.yml to push to quay
- [x] Create required make tasks to deploy

@dtan4 
I don't know how to set the image tag, because I don't know much about Docker.
We might want to invoke `v0.2.0` or something instead of `latest`.
Then should I use `$(VERSION)` instead of `latest`?
Maybe googling for a while could lead me there, but I ask you before I get drown in the ocean...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.4
```
Comment (dtan4) [inline Dockerfile]: Latest release is `alpine:3.5`.

**Review by dtan4 — CHANGES_REQUESTED** [review]

**Comment by dtan4** [comment]
> Then should I use $(VERSION) instead of latest?

I think that both `latest` and `$(VERSION)` should be prepared. `latest` points to the latest `$(VERSION)`.

**Comment by dtan4** [comment]
By the way, I don't recommend to build Docker image in this repository together, because CI build time will be so long...
[k8sec](https://github.com/dtan4/k8sec) took about 20 minutes until tagged build had finished in the same environment.

I recommend to separate repository, e.g. `dockerfile-developers-account-mapper`. 
See this: https://github.com/dtan4/dockerfile-k8sec

File: Dockerfile (hunk 1 above)
Comment (potsbo) [reply Dockerfile]: Thanks

**Comment by potsbo** [comment]
Do we have to separate repositories?
Just using quay build triggers doesn't solve the slow building issue?

**Comment by dtan4** [comment]
> Just using quay build triggers doesn't solve the slow building issue?

To use builder in Quay.io, we have to build binary or download binary in`Dockerfile`. This is not so good way...

**Comment by potsbo** [comment]
> we have to build binary or download binary inDockerfile

I see that point.

But, since this project will not be updated on a daily basis, maybe taking a while to build in ci is not a problem for us.
How about considering to separate repos later, when it becomes a problem.

> I think that both latest and $(VERSION) should be prepared. latest points to the latest $(VERSION).

Do I have to push twice in two tags?

**Comment by dtan4** [comment]
OK, let's keep single repository now.

> Do I have to push twice in two tags?

Yes. `DOCKER_IMAGE_TAG=$TRAVIS_TAG make ci-docker-release` would work fine.

**Comment by potsbo** [comment]
Added.

**Comment by dtan4** [comment]
LGTM 👍 
Are you sure that both `DOCKER_QUAY_USERNAME` and `DOCKER_QUAY_PASSWORD` were set?

**Comment by potsbo** [comment]
Yes, It's been set.
screen shot 2017-01-25 at 0 10 35

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
OK, Now I merge this.

**Pull request merged**